docs/aws/GettingStarted/FirmwareConfiguration.rst

```diff
@@ -5,85 +5,61 @@ Configure the firmware
 
 To configure the :ref:`firmware <firmware-aws-index>`, complete the following mandatory steps:
 
-.. only:: v1_5_x
-
-   1. Run the following command to print the MQTT endpoint to which your devices will connect:
+1. Run the following command to print the MQTT endpoint to which your devices will connect:
 
    .. code-block:: bash
 
       node cli info -o mqttEndpoint
 
-   #. Use this MQTT endpoint value as the value for the ``CONFIG_AWS_IOT_BROKER_HOST_NAME`` configuration variable.
-
-   #. Use ``42`` as the value for ``CONFIG_AWS_IOT_SEC_TAG``.
-
-.. only:: not v1_5_x
+#. Use this MQTT endpoint value as the value for the ``CONFIG_AWS_IOT_BROKER_HOST_NAME`` configuration variable.
 
-   1. Run the following command to print the MQTT endpoint to which your devices will connect:
+#. Use ``42`` as the value for ``CONFIG_AWS_IOT_SEC_TAG``.
 
-      .. code-block:: bash
+#. Comment in the remaining AWS settings in the :file:`prj.conf` file:
 
-         node cli info -o mqttEndpoint
+   .. code-block:: bash
+      
+      CONFIG_AWS_IOT=y
+      CONFIG_AWS_IOT_APP_SUBSCRIPTION_LIST_COUNT=1
+      CONFIG_AWS_IOT_AUTO_DEVICE_SHADOW_REQUEST=y
+      CONFIG_AWS_IOT_CLIENT_ID_APP=y
+      CONFIG_AWS_IOT_MQTT_PAYLOAD_BUFFER_LEN=2048
+      CONFIG_AWS_IOT_MQTT_RX_TX_BUFFER_LEN=2048
+      CONFIG_AWS_IOT_TOPIC_GET_REJECTED_SUBSCRIBE=y
+      CONFIG_AWS_IOT_TOPIC_UPDATE_DELTA_SUBSCRIBE=y
+      CONFIG_AWS_FOTA=y
+   
+#. Comment in the AWS logging settings in the :file:`overlay-debug.conf` file:
 
-   #. Use this MQTT endpoint value as the value for the ``CONFIG_AWS_IOT_BROKER_HOST_NAME`` configuration variable.
+   .. code-block:: bash
+   
+      CONFIG_AWS_FOTA_LOG_LEVEL_DBG=y
+      CONFIG_AWS_IOT_LOG_LEVEL_DBG=y
+      CONFIG_AWS_JOBS_LOG_LEVEL_DBG=y
 
-   #. Use ``42`` as the value for ``CONFIG_AWS_IOT_SEC_TAG``.
+#. Comment out the nRF Connect for Cloud settings in the :file:`prj.conf` file:
```

Comment:
Isn't it nRF Cloud now?

Reply:
Yes, but this PR purely focuses on separating the code that is in the v1.6.x branch. We can then backport changes.